packages/cli/src/databases/migrations/common/1723627610222-CreateInvalidAuthTokenTable.ts

```diff
@@ -0,0 +1,15 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+const tableName = 'invalid_auth_token';
+
+export class CreateInvalidAuthTokenTable1723627610222 implements ReversibleMigration {
+	async up({ schemaBuilder: { createTable, column } }: MigrationContext) {
+		await createTable(tableName)
+			.withColumns(column('token').varchar(512).primary, column('expiresAt').timestamp().notNull)
+			.withIndexOn('token', true);
```

Comment:
Both sqlite and postgres create a unique index for every PK automatically, so this creates a duplicate index. No idea about mysql.

Sqlite:
![image](https://github.com/user-attachments/assets/4bb8a994-5abf-46cd-8025-4894ff7c43c3)


Postgres:
![image](https://github.com/user-attachments/assets/9967e78f-3f26-4de0-834c-72001171d9c6)